### PR TITLE
Implement RestAPI server with Django

### PR DESCRIPTION
**Pull request by lijm1358**
This revision includes:

### References
1. https://docs.djangoproject.com/en/3.2/
2. https://www.django-rest-framework.org/

### CODE REVIEW

**Review by hyunyunV — APPROVED** [review]
LGTM

File: server/MapGo_server/urls.py [hunk 1]
```diff
@@ -13,9 +13,25 @@
...
+
+router2 = routers.DefaultRouter()
+router2.register('Mapgo/recommend', STCViewSet, basename='recommend')
+
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', include(router.urls)),
+    path('Mapgo/checkin/<str:userid>/',
+         CheckinViewSet.as_view({'delete': 'delete'})),
+    path('', include(router2.urls)),
```
Comment (Snowapril) [inline server/MapGo_server/urls.py:36]: @lijm1358 `router1`, `router2` namings are seems little bit ambiguous to me.
How about use dictionary?
```suggestion
routers = {
    "checkin" : routers.DefaultRouter()
    "recommend" : routers.DefaultRouter()
}

routers["checkin"].register('Mapgo/checkin', CheckinViewSet, basename="checkin"),
routers["recommend"].register('Mapgo/recommend', STCViewSet, basename='recommend')

urlpatterns = [
    path('admin/', admin.site.urls),
    path('', include(routers["checkin"].urls)),
    path('Mapgo/checkin/<str:userid>/',
         CheckinViewSet.as_view({'delete': 'delete'})),
    path('', include(routers["recommend"].urls)),
```

Comment (Snowapril) [inline server/MapGo_server/urls.py]: @lijm1358 `router1`, `router2` namings are seems little bit ambiguous to me.
How about use dictionary?
```suggestion
routers = {
    "checkin" : routers.DefaultRouter()
    "recommend" : routers.DefaultRouter()
}

routers["checkin"].register('Mapgo/checkin', CheckinViewSet, basename="checkin"),
routers["recommend"].register('Mapgo/recommend', STCViewSet, basename='recommend')

urlpatterns = [
    path('admin/', admin.site.urls),
    path('', include(routers["checkin"].urls)),
    path('Mapgo/checkin/<str:userid>/',
         CheckinViewSet.as_view({'delete': 'delete'})),
    path('', include(routers["recommend"].urls)),
```

Comment (lijm1358) [reply server/MapGo_server/urls.py]: Yep, this looks much better. I'll commit this change after refactoring. Thanks for reviewing!

**Review by Snowapril — APPROVED** [review]
Now looks good to me. I think `recommend1` and `recommend2` namings in several files are looks ambiguous too, but It is better to pass them to next refactoring branch. Thanks for contribution!

**Review by dldks321 — APPROVED** [review]
LGTM

**Pull request merged**